### PR TITLE
Added an exception when resource is not found

### PR DESCRIPTION
**Pull request by briemens**

### CODE REVIEW

**Comment by briemens** [comment]
Relates to issue https://github.com/krakenjs/engine-munger/issues/22 as well

**Comment by aredridel** [comment]
Awesome! I'm thinking about this, and the knowledge that it's `.properties` is just assumed, but actually dependent on other modules. I may have to think if there's a better way to do this. I don't think the packages are factored that well because of things like this.

**Comment by aredridel** [comment]
Build failure is not your fault, I'm tightening the dustjs versions to account for 2.7.0

**Comment by briemens** [comment]
The code is overcomplicated and hard to follow!

**Comment by briemens** [comment]
@aredridel The errors are not, but I see an indention issue

**Comment by aredridel** [comment]
Oh, this code is awful. I'm killing it in new content work -- adaro directly will support a lot of this. As will dust itself.

**Comment by aredridel** [comment]
Also, if you want to preserve your brain, don't look at anything in adaro labeled 'patch'.

**Comment by briemens** [comment]
@aredridel He he! :-)

**Comment by aredridel** [comment]
After actually testing with a working test suite (Thanks, Dust 2.7!), this doesn't work -- it breaks the specialization test cases, particularly because of how properties files resolve in that case. Properties not existing is fine; it's the template not loading that's the bigger problem, which this doesn't actually address.
